docs/Dialects/krnl.md

```diff
@@ -519,41 +519,84 @@ Matmul operation for a single pannel.
 Syntax:
 
 ```
-operation ::= `krnl.matmul` $A `[` $aMemStart `]` `,`
-              $B `[` $bMemStart `]` `,`
-              $C `[` $cMemStart `]` `,`
+operation ::= `krnl.matmul` $A `[` $aGlobalIndexMemStart `]` `,`
+              $B `[` $bGlobalIndexMemStart `]` `,`
+              $C `[` $cGlobalIndexMemStart `]` `,`
```

Comment:
renaming was done to better indicates which variables are global values as opposed to relative to a tile.